db/schema.rb

```diff
@@ -659,6 +659,15 @@
   add_index "organisation_visits", ["organisation_id"], name: "index_organisation_visits_on_organisation_id", using: :btree
   add_index "organisation_visits", ["visit_id", "organisation_id"], name: "index_organisation_visits_on_visit_id_and_organisation_id", unique: true, using: :btree
 
+  create_table "specifics", force: :cascade do |t|
```

Comment:
After having a ponder, I think I've landed on 'why take away options?' Tough to future imagine what the best solution will be, but it doesn't have to be one or the other, and I think there will definitely be cases where a shared model will be more effective than a database table, and vice versa.

Like, even the fact that we're respectfully clashing a bit over these two things is telling me 'aw, some authors will want to touch the schema, while others will prefer a more JSON-y approach, and that's cool.'